packages/eslint-plugin/src/rules/emotion-to-compiled/index.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Rule } from 'eslint';
+
+import type { ImportSpecifier, ImportDeclaration } from 'estree';
+
+const hasStyledImport = (node: ImportDeclaration) => node.source.value === '@emotion/styled';
+const hasCoreImport = (node: ImportDeclaration) => node.source.value === '@emotion/core';
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      recommended: true,
+    },
+    fixable: 'code',
+    type: 'problem',
+    messages: {
+      noStyled: `The '@emotion/styled' library should no longer be in use. Use '@compiled/react'.`,
```

Comment:
nit: This error message seems loose. Could we tighten it up?

> `@emotion` should not be used, use `@compiled/react` in its place.

Reply:
Agree, though:

> in its place.

to

> instead.

is more common